ecs-logs-collector.sh

```diff
@@ -328,7 +328,7 @@ get_kernel_logs() {
   if [ -e "/var/log/dmesg" ]; then
     cp -f /var/log/dmesg "$dstdir/dmesg.boot"
   fi
-  dmesg > "$dstdir/dmesg.current"
+  dmesg -T > "$dstdir/dmesg.current"
```

Comment:
If you have a chance to make a quick change: it would be more desirable to have the option fully typed out here for others who read this (eg: `--ctime`). Regardless, this certainly helps correlate events from other logs. Thanks!

Reply:
I'm using this script for automation, and I saw some other guys using it too to pull data from ECS to push in ELK and other solutions so changing the file may create breaking chains, so I did a similar pull request for.  Preventing such breaking chain.

https://github.com/awslabs/ecs-logs-collector/pull/39

Reply:
@hprudent thanks for the suggestion, if this change is going to break others using this for their diagnostics, I would prefer to not break them.